aws/sqs_event_manager/sqs_event_manager/app.py

```diff
@@ -177,3 +177,78 @@ def process_message(record: Dict) -> Dict[str, Union[str, bool]]:
         result['status'] = f'{type(error).__name__}: {error}'
         logger.error(result['status'])
     return result
+
+
+def subscription_success(message: AWSSNSMessage) -> Dict:
+    """
+    Notify SCC on subscription successfully processed
+    """
+    request_data = basic_request(message)
+    request_data['offerIdentifier'] = message.offer_id
+    request_data['isFreeTrialTermPresent'] = message.free_trial_term_present
+    return request_data
+
+
+def subscription_removed(message: AWSSNSMessage) -> Dict:
+    """
+    Notify SCC on subscription successfully removed
+    """
+    request_data = basic_request(message)
+    request_data['offerIdentifier'] = message.offer_id
+    request_data['isFreeTrialTermPresent'] = message.free_trial_term_present
+    return request_data
+
+
+def subscription_failed(message: AWSSNSMessage) -> Dict:
```

Comment:
Could these three be just one function? They seem to all do the same thing.

Reply:
I have added this duplication intentionally because I still have no 100% clarity if the request sent to SCC will really stay the same for all actions. In case it needs to change I have a method for each of them. In case they are all the same it will be a simple refactor in the future